modules/openxr/extensions/openxr_extension_wrapper_extension.h

```diff
@@ -39,7 +39,7 @@
 #include "core/os/thread_safe.h"
 #include "core/variant/native_ptr.h"
 
-class OpenXRExtensionWrapperExtension : public Object, OpenXRExtensionWrapper {
+class OpenXRExtensionWrapperExtension : public Object, public OpenXRExtensionWrapper, public OpenXRCompositionLayerProvider {
```

Comment:
We could make a new extension class, however, nothing special is done with an `OpenXRCompositionLayerProvider` until you call `openxr_api->register_composition_layer_provider()`, so I think it's safe for them to always be `OpenXRCompositionLayerProvider`, since only the extensions that really need to do anything for that will register themselves.